rosdep/base.yaml

```diff
@@ -1977,6 +1977,22 @@ libdxflib-dev:
   debian: [libdxflib-dev]
   fedora: [libdxflib-devel]
   ubuntu: [libdxflib-dev]
+libedit:
+  arch: [libedit]
+  debian: [libedit-dev]
```

Comment:
For debian, ubuntu, and rhel, the `libedit` key should be the name of the library, not the devel version of the library.  That is, for debian and ubuntu it looks like it should be `libedit2`, and for rhel it should just be `libedit`.

Reply:
fixed !

Reply:
But it show that there is an issue with libreadline (ubuntu and debian says libreadline-dev for the non-dev version, it should be libreadline5)

Reply:
> fixed !

Great, thanks.

> But it show that there is an issue with libreadline (ubuntu and debian says libreadline-dev for the non-dev version, it should be libreadline5)

Excellent point, that seems like a bug.

The problem with that now is that there are people depending on that key, and getting more than they asked for.  If we fix it, then some stuff is probably going to start failing.  @tfoote What's our backwards-compatibility policy here?